### PR TITLE
Move home screen into showcase

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Depends on https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/322

**Comment by cristianonicolai** [comment]
@nmirasch could you please take a look? :)

**Comment by nmirasch** [comment]
needed rebase here.

**Comment by cristianonicolai** [comment]
@nmirasch done :)

File: jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/client/screens/HomeViewImpl.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
```
Comment (nmirasch) [inline jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/client/screens/HomeViewImpl.java]: maybe is a good moment to fix the some home page links replacing  "Tasks List" -> "DataSet Tasks"  and "Process Instances" -> "DataSet Process Instances With Variables" and "Process Definition List" ->"Process Definitions".

Comment (nmirasch) [inline jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/client/screens/HomeViewImpl.java]: @cristianonicolai what do you think?

Comment (cristianonicolai) [inline jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/client/screens/HomeViewImpl.java]: @nmirasch sure, makes sense to me to fix the links, let me review this PR.

**Pull request merged**